units/src/amount.rs

```diff
@@ -281,6 +281,8 @@ impl std::error::Error for ParseAmountError {
 /// Returned when a parsed amount is too big or too small.
 #[derive(Debug, Copy, Clone, Eq, PartialEq)]
 pub struct OutOfRangeError {
+    // This boolean represents whether the return type is signed i.e., is a `SignedAmount` vs unsigned `Amount`.
+    // WARNING: It does not mean that we tried to parse a negative sat value into an `Amount`.
```

Comment:
Why not change this to a docstring?

```suggestion
    /// This boolean represents whether the return type is signed i.e., is a `SignedAmount` vs unsigned `Amount`.
    /// WARNING: It does not mean that we tried to parse a negative sat value into an `Amount`.
```

Reply:
Can do, thanks.